lib/utils/format.js

```diff
@@ -0,0 +1,14 @@
+export default (function() {
```

Comment:
Get rid of the export here and I'll merge, I'll add the exception to ESLint to allow string prototype extensions